### PR TITLE
Translated the accelerate.md file of the documentation to Chinese

### PR DESCRIPTION
**Pull request by liteli1987gmail**
What does this PR do?

🍅Add zh (Chinese) translation  accelerate.md
😊Add accelerate to _toctree.yml

- issue Transformers documentation translation to Chinese (Simplified) #20095

Who can review?
Documentation: @stevhliu and @MKhalusova
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by ArthurZucker — COMMENTED** [review]
Hey! Thanks a lot for contributing. 
Could you make sure your pr has a similar format as  #26002 😉 feel free to ping @stevhliu once your PR is ready for review!

**Comment by liteli1987gmail** [comment]
What does this PR do?

🍅Add zh (Chinese) translation  accelerate.md
😊Add accelerate to _toctree.yml

- issue Transformers documentation translation to Chinese (Simplified) #20095

Who can review?
Documentation: @stevhliu and @MKhalusova
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

File: docs/source/zh/accelerate.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<!--版权2022年HuggingFace团队保留所有权利。
```
Comment (ArthurZucker) [inline docs/source/zh/accelerate.md:1]:
```suggestion
<!--版权2023年HuggingFace团队保留所有权利。
```

**Review by ArthurZucker — COMMENTED** [review]
Hey! Again, could you make sure this pull request follows the correct format? 
- the title should start with " 🌐 [i18n- "  followed by which version of chinese this is (simplified or not)
- include the checks to make sure you followed the contribution guidelines for documentation! 
otherwise thanks for the contribution! 🤗

**Review by stevhliu — APPROVED** [review]
Thanks for your contribution! 🤗

**Comment by stevhliu** [comment]
Hi @liteli1987gmail, hope you don't mind me committing Arthur's suggestion for you so we can merge this PR! Thanks again for the translation :)

**Pull request merged**